runtime/lua/vim/lsp/util.lua

```diff
@@ -2328,6 +2328,88 @@ function M._refresh(method, opts)
   end
 end
 
+---@param feature string
+---@param client_id? integer
+local function make_enable_var(feature, client_id)
+  return ('_lsp_%s%s_enabled'):format(
+    feature,
+    client_id and ('_client_%d'):format(client_id) or nil
```

Comment:
`nil` will print "nil", should it be empty something like "allclients" ?
```suggestion
    client_id and ('_client_%d'):format(client_id) or "allclients"
```